frontend/src/assets/content/mthood/cutting-instructions/helpful-information.md

```diff
@@ -1,6 +1,6 @@
 * Tools you might want to consider bringing with you include a handsaw to cut your tree; gloves to protect your hands; a tarp to sit on and/or to move your tree once it's cut; and rope or straps to secure your tree to your vehicle.
-* Be sure another tree has space to grow within 8 feet of the one you plan to cut. Choosing a tree from an overcrowded area will give the remaining trees more space to grow.
-* Be aware of where the tree will fall and ensure all members of your party (and your vehicle) are out of the path of the falling tree.
+* Be sure another tree has space to grow within 8 feet of the one you plan to cut. Choosing a tree from an dense forested area will give the remaining trees more space to grow.
+* Be aware of where the tree will fall and ensure all people and vehicles are out of the path of the falling tree.
```

Comment:
I think you might want to add `please` before the ensure because it currently reads that the try will ensure that people are out of the way.

Reply:
We could do that if it's easy enough to add - I don't think we say please  on any of the other pages so if we add it here, we should add it to the other forests' pages as well for consistency sake.